contentcuration/contentcuration/frontend/shared/constants/ContentKinds.js

```diff
@@ -0,0 +1,26 @@
+export default [
+  {
+    kind: 'document',
+  },
+  {
+    kind: 'slideshow',
+  },
+  {
+    kind: 'topic',
+  },
+  {
+    kind: 'video',
+  },
+  {
+    kind: 'html5',
+  },
+  {
+    kind: 'h5p',
+  },
+  {
+    kind: 'audio',
+  },
+  {
+    kind: 'exercise',
+  },
+];
```

Comment:
There are many changes in this PR that I would not have expected to be associated with adding Storybook. Was there perhaps a rebase or merge issue?

Reply:
No, as mentioned in the PR description, I needed to update Babel and that showed problems with some of the dependencies between files using mixed CommonJS and ES6 modules. I would recommend checking this PR commit by commit - all work is separated into steps.

Reply:
The file you pointed out is part of constants and it's been copied beside related files to `/frontend` so I can reference it as ES6 module from there. The original files stayed CommonJS only so we can continue to reference them from old Backbone parts that will be removed over time completely.

It's basically just cleanup work that would need to be done anyway and allowed me to get rid of mixed modules https://github.com/learningequality/studio/pull/1805/commits/ebf6b30c73f9cc85699a6007ac596467c66334a5

Reply:
It's quite simple though maybe it's one of those things difficult to explain in letters :) I've invited more folks because I am not sure who'd like to review - if you're interested maybe we can plan a call and go through that?

Reply:
okay thanks - will review more carefully